Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -5411,9 +5411,96 @@ RefPtr<StyleValue> Parser::parse_grid_track_sizes(Vector<ComponentValue> const&
 {
     Vector<CSS::GridTrackSize> params;
     for (auto const& component_value : component_values) {
-        // FIXME: Incomplete as a GridTrackSize can be a function like minmax(min, max), etc.
         if (component_value.is_function()) {
-            params.append(Length::make_auto());
+            auto const& function_token = component_value.function();
+            // https://drafts.csswg.org/css-grid/#repeat-syntax
```

Comment:
Try to use w3.org spec links if possible. The csswg drafts change more often, and half the time the website doesn't work properly. :sweat_smile: For example, https://www.w3.org/TR/css-grid-1/#repeat-syntax or https://www.w3.org/TR/css-grid-2/#repeat-syntax depending on which level you plan on implementing. (The difference seems to be that Level 2 adds subgrids.)

Reply:
Alright! I hadn't made that distinction, I'll be sure to use w3.org going forward